src/system/SystemConfig.h

```diff
@@ -358,6 +358,26 @@
 #define CHIP_SYSTEM_CONFIG_PACKETBUFFER_POOL_SIZE 15
 #endif /* CHIP_SYSTEM_CONFIG_PACKETBUFFER_POOL_SIZE */
 
+/**
+ *  @def CHIP_CONFIG_TCP_SUPPORT_CLIENT
+ *
+ *  @brief
+ *      Indicates whether the current node supports TCP Client
+ */
+#ifndef CHIP_CONFIG_TCP_SUPPORT_CLIENT
+#define CHIP_CONFIG_TCP_SUPPORT_CLIENT 0
```

Comment:
Doesn't it more belong to `InetConfig.h` than `SystemConfig.h`? And what's the relationship between these and `INET_CONFIG_ENABLE_TCP_ENDPOINT`?

Reply:
> And what's the relationship between these and INET_CONFIG_ENABLE_TCP_ENDPOINT?

Either of these probably requires INET_CONFIG_ENABLE_TCP_ENDPOINT, but you could support that without claiming either client or server TCP support....